File: Python/Dynamic_Variables.py
obj = {}
def makeDynamic(str, data):
    obj[str] = data
    print obj

def addTo(str, data):
    obj[str] += data
    print obj

def printMe(str):
    print obj[str]
    
#makes a dynamic Dictionary
#adds data
#and then prints it

File: WebBuilding/Javascript/Dynamic_Variables.js
//code that allows for dynamic Variables - with unique names - to be made

var obj = {};
//==========================
function makeDynamic(str, data){
    obj[str] = data;
    return obj;
}

function addData(str, data2){
    obj[str] += 'hello';
    console.log('obj', obj); 
}
